<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="MAY40" name="com.qa.test.LoginTest" tests="8" failures="0" timestamp="2020-09-12T13:48:49 GMT+07:00" time="0.011" errors="2">
  <testcase name="successLogin" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- successLogin -->
  <system-out/>
  <testcase name="beforeTest" time="0.010" classname="com.qa.test.LoginTest">
    <error type="java.lang.Error" message="Unresolved compilation problem: 
	The local variable url may not have been initialized
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
The local variable url may not have been initialized

at com.qa.BaseTest.beforeTest(BaseTest.java:217)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <system-out/>
  <testcase name="successLogin" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- successLogin -->
  <system-out/>
  <testcase name="beforeTest" time="0.001" classname="com.qa.test.LoginTest">
    <error type="java.lang.Error" message="Unresolved compilation problem: 
	The local variable url may not have been initialized
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
The local variable url may not have been initialized

at com.qa.BaseTest.beforeTest(BaseTest.java:217)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <system-out/>
  <testcase name="invaildUser" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- invaildUser -->
  <system-out/>
  <testcase name="invaildPassword" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- invaildPassword -->
  <system-out/>
  <testcase name="invaildUser" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- invaildUser -->
  <system-out/>
  <testcase name="invaildPassword" time="0.000" classname="com.qa.test.LoginTest">
    <skipped/>
  </testcase> <!-- invaildPassword -->
  <system-out/>
</testsuite> <!-- com.qa.test.LoginTest -->
